src/irc/IrcEventBroker.ts

```diff
@@ -564,4 +582,9 @@ export class IrcEventBroker {
             await req();
         })();
     }
+
+    static isValidNick(nick: string) {
+        // The first character must be one of these.
+        return /^[A-Za-z\[\]\\`_^\{\|\}]/.test(nick[0])
```

Comment:
((we don't enforce lines to end on a semicolon?))

Reply:
strange. will fix.